cluster/libvirt-coreos/util.sh

```diff
@@ -279,6 +285,51 @@ function kube-up {
   echo
   echo "You can control the Kubernetes cluster with: 'cluster/kubectl.sh'"
   echo "You can connect on the master with: 'ssh core@${KUBE_MASTER_IP}'"
+
+  wait-registry-readiness
+
+}
+
+function create_registry_rc() {
+  echo " Create registry replication controller"
+  local kubectl="${KUBE_ROOT}/cluster/kubectl.sh"
+  "${kubectl}" create -f $ROOT/registry-rc.yaml
+  local timeout=120
+  while [[ $timeout -ne 0 ]]; do
+    phase=$("${kubectl}" get pods -n kube-system -lk8s-app=kube-registry --output='jsonpath={.items..status.phase}')
+    if [ "$phase" = "Running" ]; then
+      return 0
+    fi
+    timeout=$(($timeout-1))
+    sleep .5
+  done
+}
+
+
+function create_registry_svc() {
+  echo " Create registry service"
+  local kubectl="${KUBE_ROOT}/cluster/kubectl.sh"
+  "${kubectl}" create -f "${KUBE_ROOT}/cluster/addons/registry/registry-svc.yaml"
+}
+
+function wait-registry-readiness() {
+  if [[ "$ENABLE_CLUSTER_REGISTRY" == "true" ]]; then
```

Comment:
I'd prefer the "go-style" here where you check for != true and return and leave the rest of the function de-dented by 1 level.